salt/modules/boto_cloudtrail.py

```diff
@@ -483,7 +487,8 @@ def list_tags(Name,
 
     try:
         conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
-        rid = _get_trail_arn(Name)
+        rid = _get_trail_arn(Name,
+                             region=None, key=None, keyid=None, profile=None)
```

Comment:
Is there a reason why these values are all `None`, instead of `region=region, key=key, ...`?